pom.xml

```diff
@@ -44,6 +44,7 @@
   <properties>
     <jenkins.version>2.176.4</jenkins.version>
     <java.level>8</java.level>
+    <useBeta>true</useBeta>
```

Comment:
@timja @oleg-nenashev I don't know what it does :-/

Reply:
the API is marked as beta, https://github.com/jenkinsci/extended-read-permission-plugin/blob/master/src/main/java/hudson/plugins/extendedread/SystemReadPermission.java#L14

you need to opt-in to using it, so that you don't accidentally end up using beta APIs.

Reply:
AHHHHHH Thanks :-)